integration-tests/api/__tests__/admin/price-list.js

```diff
@@ -1112,12 +1112,6 @@ describe("/admin/price-lists", () => {
         expect.objectContaining({
           id: "test-prod-2",
           variants: expect.arrayContaining([
-            expect.objectContaining({
```

Comment:
self-review: the test was wrong in develop and now it does not pass which is expected. So the changes are aligned with the real expected result